scss-lint.yml

```diff
@@ -61,7 +61,7 @@ linters:
   ImportPath:
     enabled: true
     leading_underscore: false
-    filename_extension: true
+    filename_extension: false
```

Comment:
We shouldn't have activated this rule, in the first place, since it was a problem of the generator, not a linting mistake.
This is also the default behavior: https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#importpath